File: translations/codenet/deepseek_commented/C++2P/deepseek/Code/s150958471.py
# Include necessary headers
# Python does not require headers for input and output operations.

# Declare arrays
# Two arrays, `train` and `bus`, are declared to store the values.
train = [0] * 2
bus = [0] * 2

# Input values
# The values for `train` and `bus` are taken from the user.
train[0], train[1] = map(int, input().split())
bus[0], bus[1] = map(int, input().split())

# Calculate the sum
# The code then calculates the sum of the minimum values from both arrays.
result = min(train[0], train[1]) + min(bus[0], bus[1])

# Output the result
# Finally, the result is printed to the console.
print(result)

# End-of-code comment

File: translations/codenet/deepseek_commented/C++2J/deepseek/Code/s150958471.java
public class Main {
    public static void main(String[] args) {
        // Declare arrays
        int[] train = new int[2];
        int[] bus = new int[2];

        // Input values
        // The values for `train` and `bus` are taken from the user.
        java.util.Scanner scanner = new java.util.Scanner(System.in);
        train[0] = scanner.nextInt();
        train[1] = scanner.nextInt();
        bus[0] = scanner.nextInt();
        bus[1] = scanner.nextInt();

        // Calculate the sum
        // The code then calculates the sum of the minimum values from both arrays.
        int result = Math.min(train[0], train[1]) + Math.min(bus[0], bus[1]);

        // Output the result
        // Finally, the result is printed to the console.
        System.out.println(result);
    }
}
